source/includes/_simulator-guide.html.md

```diff
@@ -0,0 +1,130 @@
+# Simulator Training
+
+> ![Simulator Diagram](../images/brain-sim-marketing.png)
+
+This guide is intended to give you information and best practices you’ll need to train a BRAIN on The Bonsai Platform with simulators. It covers best practices such as running simulations in parallel, general guidelines like how to use corporate proxies with the CLI, and other tips pertaining to simulators and simulations.
+
+Currently, the Bonsai Platform only uses simulators for applied deep reinforcement learning (DRL). For more background on applied DRL with simulations read our blog post on [simulations as a training environment][6].
+
+We have simple coordinators for Python & C++, two common languages used in systems modeling. We also have coordinators for Simulink, EnergyPlus and Gazebo. To learn more about connecting your simulator, refer to our [Library Reference][4].
```

Comment:
reader qs: what's a coordinator? why do I need it? 

Victor q: do you mean SDK or coordinator for python and C++?